pkg/plugin/packetparser/packetparser_linux.go

```diff
@@ -207,22 +208,28 @@ func (p *packetParser) Start(ctx context.Context) error {
 		p.callbackID = ps.Subscribe(common.PubSubEndpoints, &fn)
 	}
 
-	outgoingLinks, err := utils.GetDefaultOutgoingLinks()
-	if err != nil {
-		return err
-	}
-	if len(outgoingLinks) == 0 {
-		return errNoOutgoingLinks
-	}
-	outgoingLink := outgoingLinks[0] // Take first link until multi-link support is implemented
+	if p.cfg.DataAggregationLevel == kcfg.High {
+		p.l.Info("Skipping attaching bpf program to default interface")
+	} else {
```

Comment:
Specifically check `DataAggregationLevel == low` for attaching `eth0`. It's very resource consuming to run the eth0 program, so I don't envision it being part of any other level.